### ISSUE TITLE
Request) Manual Select For Rename

### ISSUE THREAD

**Issue by hardknoxni69a** [Necrobot-Private/NecroBot#849]
Add a button to GUI next to transfer and pokedex where user can rename the selected pokemon

**Comment by jjskuld** [comment]
You can click the name of the pokemon to enter edit mode and enter a new name for the pokemon.

**Comment by hardknoxni69a** [comment]
Yes I use this feature often but im saying we can select multiple pokemon
say certain 60 pokemon and click apply rename formula and all those
selected will be renamed

**Comment by hardknoxni69a** [comment]
I can't leave auto rename on because it renames pokemon that I have changed
to something besides available formula. Or even if the nickname would only
be applied if it wasn't custom so if I had "pidgey" it would rename to
100/30 or then I level it up it would rename to 100/31 but if I named it
"airbud" it would skip it because it is custom and doesn't fit with formula

**Comment by hardknoxni69a** [comment]
@jjskuld you seem real good with adding stuff like this i know we have the transfer all selected button but could we get a rename all selected button as well? select xyz then hit button and only those will be renamed

**Comment by SLASHLogin** [comment]
O think better should be not rename selected but just skip renamed by us pokemons. 
So when I rename through WIN pidgey to "pid" then add Pokemon's ID to list "don't change name". You know what I'm thinking about?

**Comment by hardknoxni69a** [comment]
That method would probably work as well so it doesn't rename the Pokemon u
selected just not sure which would be easier to implement

**Comment by SLASHLogin** [comment]
That's first which one is easier to implement. 
And second I think You just want to protect names of your poks,but if you have in console set rename that it should change others pokemons name automatically without "command" (this button). 
Or we can make check box or something like that to show which one we want protect from rename and maybe when cursor is on the box it shows message (just like options descr.) "don't change name".
